### PR TITLE
Fixes #3323 : error with currencies in multi-company environnement

### PR DESCRIPTION
**Pull request by alexis-via**
Please refer to my code analysis on the bug report #3323 on odoo/odoo

### CODE REVIEW

File: addons/account/report/account_invoice_report.py [hunk 1]
```diff
@@ -36,8 +36,13 @@ def _compute_amounts_in_user_currency(self, cr, uid, ids, field_names, args, con
             context={}
         currency_obj = self.pool.get('res.currency')
         currency_rate_obj = self.pool.get('res.currency.rate')
-        user_currency_id = self.pool.get('res.users').browse(cr, uid, uid, context=context).company_id.currency_id.id
-        currency_rate_id = currency_rate_obj.search(cr, uid, [('rate', '=', 1)], limit=1, context=context)[0]
+        user = self.pool.get('res.users').browse(cr, uid, uid, context=context)
```
Comment (pedrobaeza) [inline addons/account/report/account_invoice_report.py:39]: I think you can use `self.env.user` although you use old API.

**Comment by alexis-via** [comment]
@pedrobaeza You are certainly right, self.env.user would be cleaner. I just wanted to stay as close as possible to the original code in order to maximize my chances to see my PR accepted in the official odoo branch (I know OCA guys prefer modern code style, but OpenERP S.A. reviewers prefer to minimize the changes)

**Comment by pedrobaeza** [comment]
OK then. Thanks.

:+1:

**Comment by alexis-via** [comment]
Note that the Travis error is NOT linked to my changes (No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)

**Comment by sebastienbeau** [comment]
This bug have been merged in odoo8 here https://github.com/odoo/odoo/commit/45485fe1d67f8a95e043b076da29951c2245fe41. So I close the PR

**Pull request closed without merging**